libcflib/whoosh/fields.py

```diff
@@ -53,7 +53,7 @@ def add(self, name, fieldtype, glob=False):
         if type(fieldtype) is type:
             try:
                 fieldtype = fieldtype()
-            except:
+            except Exception:
                 e = sys.exc_info()[1]
```

Comment:
Would this be better as `except Exception as e`, where e is the actual exception object for parsing?

Reply:
Not sure. This way is consistent with the base whoosh Schema class.

Reply:
Ah ok